### DISCUSSION TITLE
Selección de proyecto

### DISCUSSION THREAD

**Discussion by Mariannaosorio** [rramosp/ai4eng.v1#78] [00 - General]
Miramos algunos proyectos y seleccionamos estas 5 opciones que nos parecen interesantes, le compartimos los links para que nos ayude porfavor a escoger el más adecuado para el nivel del curso.

https://www.kaggle.com/c/osic-pulmonary-fibrosis-progression/data

https://www.kaggle.com/c/lish-moa/data

https://www.kaggle.com/c/ventilator-pressure-prediction/code

https://www.kaggle.com/c/rsna-pneumonia-detection-challenge/data

https://www.kaggle.com/c/msk-redefining-cancer-treatment/data

Gracias, quedamos atentas

**Comment by rramosp** [comment]
hola marianna, cualquiera de los cuatro me parecen válidos, tienen la suficente complejidad como para que tengas una experiencia de aprendizaje. El primero y el penúltimo parten de imágenes,  que se sale un poco del ámbito del curso, pero si te le quieres medir, adelante.

en ese caso, te recomendaría que miraras el módulo cuatro de nuestro curso de deep learning que está en línea (con el mismo formato de videos y autocorrectores que este de IA)

https://rramosp.github.io/2021.deeplearning/intro.html

saludos
raúl
